EventListener/AttachRefreshTokenOnSuccessListener.php

```diff
@@ -56,7 +59,8 @@ public function attachRefreshToken(AuthenticationSuccessEvent $event)
             $datetime->modify('+'.$this->ttl.' seconds');
 
             $refreshToken = $this->refreshTokenManager->create();
-            $refreshToken->setUsername($user->getUsername());
+            $getter = 'get'.ucfirst($this->userIdentityField);
```

Comment:
@shubaivan ok, i will try it, thanks.

Reply:
@clement-michelet maybe you right, but how encapsulation don't work, because property private...
So, you can change it if needed

Reply:
I miss your point about property visibility and encapsulation. Could you explain ?

The accessor component will try to access a public property or try to find public accessor with pre-defined naming scheme like `get`, `is`, property name method, etc. It is enough flexible to not block users and avoid fatal error on unavailable method.

If the property is not accessible (i.e. not public and no accessor), it will throw an exception, It doesn't try to overrible class visibility. It's easier to catch than a Throwable about an non-existing getter.

Reply:
hmm, you right

Reply:
@clement-michelet done, could you review please last commit ?

Reply:
Ok for me !